R/dsl.R

```diff
@@ -64,3 +64,41 @@ dropdown <- function(name, choices, choices_value = choices, default_text = 'Sel
     tags$script(paste0("$('.ui.dropdown.", unique_dropdown_class, "').dropdown().dropdown('set selected', '", value,"');"))
   )
 }
+
+#' Create Semantic UI tabs
+#'
+#' This creates tabs with content using Semantic UI styles.
+#'
+#' @param tabs A list of tabs. Each tab is a list of two elements - first element defines menu item, second element defines tab content.
+#' @param id Id of the menu element (default: randomly generated id)
+#' @param menu_class Class for the menu element (default: "top attached tabular")
+#' @param tab_content_class Class for the tab content (default: "bottom attached segment")
+#'
+#' @export
+tabset <- function(tabs, id = generate_random_id(), menu_class = "top attached tabular", tab_content_class = "bottom attached segment") {
```

Comment:
It might make sense here to use named lists instead of lists, like: `list(menu = ..., content = ...)`. Than in your code you can use `.$menu` instead of `.[[1]]` which is way more readable.